src/check_list.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
```

Comment:
We shouldn't really need this. Let's not make this both module and a script, you agree? or alternatively if you think it is too early to think about that, I'm OK.